charts/verdaccio/templates/htpasswd-secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.useSecretHtpasswd }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  labels:
+    app: {{ template "verdaccio.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
Think it'd be better to use the `verdaccio.labels` util (`{{- include "verdaccio.labels" . | nindent 4 }}`) would be better than hard coding the labels